lib/output.js

```diff
@@ -67,6 +71,14 @@ module.exports = function (parameters, response) {
     threshold
   ));
 
+  if (parameters.or) {
+    console.log('\nHere are your optimized images: ', optimizedResoruceURL);
+  }
+
+  if (parameters.download) {
+    got.stream(optimizedResoruceURL).pipe(createWriteStream('optimized.zip'));
+  }
```

Comment:
`got` is writing a `zip` file which is `0kb` missing something? 
 //cc @sindresorhus